packages/web-twig/tests/components-fixtures/accordionDefault.twig

```diff
@@ -0,0 +1,32 @@
+<Accordion id="AccordionExample">
+    <AccordionItem id="AccordionItemExample0">
+        <AccordionHeader
+            id="AccordionItemExample0Header"
+            for="#AccordionItemExample0Content"
```

Comment:
for should be without hash. We add it in JS.

Reply:
You're right, I don't know how it got there in that one case. Fixed